lib/assets/javascripts/views/orgs/admin_edit.js

```diff
@@ -1,29 +1,67 @@
-// Import TinyMCE
-import tinymce from 'tinymce/tinymce';
-import { Tinymce } from '../../utils/tinymce';
 import ariatiseForm from '../../utils/ariatiseForm';
-import { SHOW_ORG_BANNER_MESSAGE } from '../../constants';
+import { Tinymce } from '../../utils/tinymce';
 
 $(() => {
   ariatiseForm({ selector: '#edit_org_details_form' });
 
-  // Returns text statistics for the specified editor by id
-  const getStats = (id) => {
-    const body = tinymce.get(id).getBody();
-    const text = tinymce.trim(body.innerText || body.textContent);
-    return {
-      chars: text.length,
-    };
+  // We only allow up to 3 URLs
+  const toggleAddUrlLink = () => {
+    if ($('#org-link-section').find('div.org-link').length >= 3) {
```

Comment:
Today is 3 links, but tomorrow could be more. Please, extract it to the constants file

Reply:
yes agreed. I also ended up pulling in the number-to-text package to update the '(Up to three)' message on the page. We will need to be able to swap in the appropriate locale though here. Maybe we can combine the handler for the one we will use for Timeago.